test/fixtures/print-delayed.js

```diff
@@ -0,0 +1,3 @@
+setTimeout(() => {
+  console.log('delayed');
+}, 1000);
```

Comment:
1s is a bit too long, don’t you think? Let’s not slow down the tests too much, I think 100ms would be plenty, we can raise it up to e.g. 300ms if you think that’s not enough. 
```suggestion
}, 100);
```

Reply:
1000 is what Joyee's test had.